build_deps/build-requirements.txt

```diff
@@ -0,0 +1 @@
+tensorflow==2.1.0
```

Comment:
Maybe we should remove that too to be able to build with tensorflow, tensorflow-cpu, tf-nightly?

Reply:
So that's a bit tricky. At the moment I believe we depend on some of the TF-GPU framework for our build (at least when compiling CUDA kernels). Certainly willing to look at a PR to change that but would rather merge this first.

Only developers should be using the build scripts so no risk of a general user running into an issue with this.

Reply:
Sure thing. If it's not trivial, let's do that later on.